datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG - datadog_checks_base
 
+## 25.6.0 / 2022-07-11
+
+* [Added] Ship `pymongo-srv` to support DNS seed connection schemas. See [#12442](https://github.com/DataDog/integrations-core/pull/12442).
+* [Fixed] Fix typo detection for config field aliases. See [#12468](https://github.com/DataDog/integrations-core/pull/12468).
+* [Fixed] [PerfCountersBaseCheck] Refresh performance objects in a separate thread. See [#12372](https://github.com/DataDog/integrations-core/pull/12372).
```

Comment:
I don't mind leaving this in because it says what base check is refreshing objects in a separate thread. Otherwise it may seem like every check is doing this.